src/main/io/osd.c

```diff
@@ -385,6 +385,20 @@ static void osdFormatMessage(char *buff, size_t size, const char *message)
     buff[size - 1] = '\0';
 }
 
+void osdSetStat(uint8_t statIndex, bool enabled)
+{
+    if (enabled) {
+        osdConfigMutable()->enabled_stats |= (1 << statIndex);
+    } else {
+        osdConfigMutable()->enabled_stats &= ~(1 << statIndex);
+    }
+}
+
+bool osdGetStat(uint8_t statIndex)
```

Comment:
Should this be `osdSetStatEnabled`, `osdGetStatEnabled`?

Reply:
The `osdSetStat()` can both enable and disable (needed for call from msp.c) so I avoided implying that it could only enable.